using_openshift/builds.adoc

```diff
@@ -78,6 +78,28 @@ valid `ref` can be a SHA1 tag or a branch name.
 where the build looks for the application source code. If your application exists inside a sub-directory,
 you can override the default location (the root folder) using this field.
 
+[[using-the-sti-environment-file]]
+== STI environment file
```

Comment:
This heading isn't rendering because the content was added within the above example (====) block, which ends on line 103.